README.md

```diff
@@ -1098,6 +1098,31 @@ console.log(invoice.errors);
 So even though the invoice object didn't have any validation errors, the `#validate` method returned
 `false` because one of its owned line item object did have an error.
 
+Transis also supports validation contexts. Multiple contextual validations can be registered for the same attribute.
+Each contextual validation will executed depending on the context provided. It's important to note that all attributes
+without a registered context will execute regardless of the context provided.
+
+```javascript
+var Invoice = Transis.Model.extend('Invoice', function() {
+  this.attr('name', 'string');
+  this.hasMany('lineItems', 'LineItem');
+
+  this.validate('name', function() {
+    if (!this.name || this.name.length < 6) {
+      this.addError('name', 'must be at least 6 characters');
+    }
+  }, {on: 'nameContext'});
+});
+
+var invoice = new Invoice({name: 'foo'});
+console.log(invoice.validate());
+// true since there is no context provided
```

Comment:
Only `true` gets logged here so I'd only show that. The rest of the examples don't have explanations in the output.